### PR TITLE
create web/mobile strategy, and first class test(move good to box)

### PR DESCRIPTION
**Pull request by TarasTsyhanko**

### CODE REVIEW

File: src/main/java/bo/BoxBusinessObject.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import elements.PageElement;
+import factory.DriverFactory;
+import org.testng.Assert;
+import page.Context;
+import page.StrategyBox;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class BoxBusinessObject {
```
Comment (MykhailoTsiganko) [inline src/main/java/bo/BoxBusinessObject.java]: rename CartAction

File: src/main/java/bo/BoxBusinessObject.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    public List<String> getAllTitleInBox() {
+        return strategy.getAllTitleInBox()
+                .stream()
+                .map(PageElement::getText)
+                .collect(Collectors.toList());
+    }
+
+    public void verifySelectedGoodIsInBox(List<String> titlesInBox, String selectedTitle) {
```
Comment (MykhailoTsiganko) [inline src/main/java/bo/BoxBusinessObject.java]: move to CartValidator

File: src/main/java/factory/DriverFactory.java [hunk 3]
```diff
@@ -15,28 +23,40 @@ private static void setWait(WebDriver driver) {
...
-    public static void initDriver() {
-        String browser = System.getProperty("browser");
-        WebDriver driver = null;
-        DriverManagerType driverType = DriverManagerType.CHROME;
-        if (browser != null && browser.equals("firefox")) {
-            driverType = DriverManagerType.FIREFOX;
-        } else if (browser != null && browser.equals("edge")) {
-            driverType = DriverManagerType.EDGE;
+    private static void initDriver() {
+        String devise = System.getProperty("devise");
```
Comment (MykhailoTsiganko) [inline src/main/java/factory/DriverFactory.java]: rename to device

File: src/main/java/factory/DriverFactory.java [hunk 4]
```diff
@@ -15,28 +23,40 @@ private static void setWait(WebDriver driver) {
...
-        } catch (ClassNotFoundException | InstantiationException | IllegalAccessException e) {
-            e.printStackTrace();
-        }
-        assert driver != null;
         driver.manage().window().maximize();
         setWait(driver);
         DRIVER = driver;
     }
 
+    private static WebDriver createWeb(String devise) {
```
Comment (MykhailoTsiganko) [inline src/main/java/factory/DriverFactory.java]: rename to create webDriver

File: src/main/java/factory/DriverFactory.java [hunk 5]
```diff
@@ -15,28 +23,40 @@ private static void setWait(WebDriver driver) {
...
+            return new FirefoxDriver();
+        } else if ("edge".equals(devise)) {
+            WebDriverManager.edgedriver().operatingSystem(OperatingSystem.WIN).setup();
+            return new EdgeDriver();
+        }
+        WebDriverManager.chromedriver().setup();
+        return new ChromeDriver();
+    }
+
+    private static WebDriver createMobile() {
```
Comment (MykhailoTsiganko) [inline src/main/java/factory/DriverFactory.java]: remove

File: src/test/java/tests/GoodsInBoxTest.java [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+public class GoodsInBoxTest {
+    private BoxBusinessObject boxBusinessObject;
+
+    @BeforeMethod
+    public void setUp(){
+        boxBusinessObject = new BoxBusinessObject();
+    }
+
+    @Test
```
Comment (MykhailoTsiganko) [inline src/test/java/tests/GoodsInBoxTest.java]: add text description

**Review by MykhailoTsiganko — CHANGES_REQUESTED** [review]

File: src/main/java/bo/BoxBusinessObject.java (hunk 1 above)
Comment (TarasTsyhanko) [reply src/main/java/bo/BoxBusinessObject.java]: fixed

File: src/main/java/bo/BoxBusinessObject.java (hunk 2 above)
Comment (TarasTsyhanko) [reply src/main/java/bo/BoxBusinessObject.java]: fixed

File: src/main/java/factory/DriverFactory.java (hunk 3 above)
Comment (TarasTsyhanko) [reply src/main/java/factory/DriverFactory.java]: fixed

File: src/main/java/factory/DriverFactory.java (hunk 4 above)
Comment (TarasTsyhanko) [reply src/main/java/factory/DriverFactory.java]: fixed

File: src/main/java/factory/DriverFactory.java (hunk 5 above)
Comment (TarasTsyhanko) [reply src/main/java/factory/DriverFactory.java]: fixed

File: src/test/java/tests/GoodsInBoxTest.java (hunk 6 above)
Comment (TarasTsyhanko) [reply src/test/java/tests/GoodsInBoxTest.java]: what you mean?
description about test?

Comment (MykhailoTsiganko) [reply src/test/java/tests/GoodsInBoxTest.java]: @Test(description="description")

Comment (TarasTsyhanko) [reply src/test/java/tests/GoodsInBoxTest.java]: fixed

**Review by MykhailoTsiganko — APPROVED** [review]

File: src/test/java/tests/GoodsInCartTest.java [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+import java.util.List;
+
+public class GoodsInCartTest {
+    private CartActions cartActions;
+    private CartValidator validator;
+
+    @BeforeMethod
+    public void setUp(){
+        cartActions = new CartActions();
```
Comment (MykhailoTsiganko) [inline src/test/java/tests/GoodsInCartTest.java:19]: use @Inject

**Pull request merged**